documentation/modules/auxiliary/scanner/http/http_put.md

```diff
@@ -7,7 +7,7 @@ This module can abuse misconfigured web servers to upload and delete web content
 2. Do: ```set RHOSTS [IP]```
 3. Do: ```set RPORT [PORT]```
 4. Do: ```set PATH [PATH]```
-5. Do: ```set FILENAME [FILNAME]```
+5. Do: ```set FILENAME [FILENAME]```
```

Comment:
I guess this is technically broken docs now, but the `Did you mean $x?` functionality should kick in if the user typos anyway

Reply:
I think you have the variable name and the identifier mixed up, this change has no change on the actual things being typed by the user

Reply:
💯